src/main/java/racingcar/model/Car.java

```diff
@@ -0,0 +1,26 @@
+package racingcar.model;
+
+public class Car {
+
+    private static final int MOVEMENT_THRESHOLD = 3;
+    public int position;
```

Comment:
```suggestion
    private static final int MOVEMENT_THRESHOLD = 3;
    
    public int position;
```
상수와 상태값의 구분을 지어주면 좋습니다 :)